bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BookieShell.java

```diff
@@ -2050,6 +2059,53 @@ protected void readLedgerIndexEntries(long ledgerId) throws IOException {
     }
 
     /**
+     * Read ledger index entries
+     * 
+     * @param ledgerId
+     * @return
+     * @throws IOException if failed to init KeyValueStorageRocksDB
+     */
+	protected int readLedgerIndexEntriesFromDbLedgerStorage(long ledgerId) throws IOException {
```

Comment:
actually `DbLedgerStorage` initializes multiple things other than `EntryLocationIndex` such as
`LedgerMetadataIndex`, `GC`, `EntryLogger`. And also, this cmd requires `EntryLocationIndex` instance which starts `rocksDB` in readOnly mode by passing custom `KeyValueStorageFactory` factory. So, I think it's bettter to keep this code in `Cmd` rather adding different init method in DbStorage just for `EntryLocationIndex` .  Any thoughts?